examples/pl_loops/yielding_training_step.py

```diff
@@ -40,7 +40,7 @@
 # code when dealing with multiple optimizers (automatic optimization).                      #
 #                                                                                           #
 # Learn more about the loop structure from the documentation:                               #
-# https://pytorch-lightning.readthedocs.io/en/stable/extensions/loops.html                  #
+# https://pytorch-lightning.readthedocs.io/en/LTS/extensions/loops.html                  #
```

Comment:
It actually doesn't exist (404). Is it correct?

Reply:
just uploaded :)